packages/perseus/src/widgets/interactive-graphs/reducer/use-graph-state.ts

```diff
@@ -0,0 +1,18 @@
+import React, {createContext} from "react";
+
+import type {InteractiveGraphAction} from "./interactive-graph-action";
+import type {InteractiveGraphState} from "../types";
+
+export const GraphStateContext = createContext<{
+    state: InteractiveGraphState;
+    dispatch: React.Dispatch<InteractiveGraphAction>;
+}>({
+    state: {} as any,
+    dispatch: () => {},
```

Comment:
TS makes me provide defaults, but this will immediately get overwritten when the provider component provides values for these.